packages/block-editor-tools/README.md

```diff
@@ -29,25 +29,29 @@ This package assumes your project is running in an environment compatible with t
 
 To use modules from this package, import them into your files using the `import` declaration
 
-```js
+```jsx
 import { usePostMeta } from '@alleyinteractive/block-editor-tools';
 
 const MyComponent = () => {
-	const [meta, setMeta] = usePostMeta();
-	const { my_meta_key: myMetaKey } = meta;
-
-	return (
-		<TextControl
-			label={__('My Meta Key', 'alley-scripts')}
-			onChange={(newValue) => setMeta({ ...meta, my_meta_key: newValue })}
-			value={myMetaKey}
-		/>
-	);
+ const [meta, setMeta] = usePostMeta();
```

Comment:
If you're going to switch tabs for spaces, that's fine, but please use 2 spaces, as that's our ESLint standard, and it's more readable that way.